script.py

```diff
@@ -42,8 +44,10 @@ def choose_spell(class_name):
         # Select random spell
         randomSpell = filteredData[random.randint(0, len(filteredData) - 1)]
         # Print random spell
+        chosenSpell = randomSpell['desc'].split(".")
         return Color.GREEN + randomSpell['name'] + Color.END + "\n" + textwrap.fill(
-            textwrap.shorten(randomSpell['desc'], width=300), width=100)
+            ((chosenSpell[0]).replace("<p>", "") + ". " + chosenSpell[1] + "."), width=100)
+
```

Comment:
This is getting pretty complex. Some comments here to describe whats going on would be really helpful

Reply:
I take it that `chosenspell[0]` is the first sentence, and `chosenSpell[1]` is the second sentence from the description? 
Also removing every instance of `<p>` from the description

To help this even further I would actually split this all up if you could instead of doing everything in the return statement. It'll also be useful for when commenting what each different part does.

Reply:
👍